decidim-elections/lib/decidim/elections/test/factories.rb

```diff
@@ -15,12 +15,25 @@
     description { Decidim::Faker::Localized.wrapped("<p>", "</p>") { generate_localized_title } }
     start_time { 1.day.from_now }
     end_time { 3.days.from_now }
+    published_at { Time.current }
     component { create(:elections_component) }
 
     trait :started do
       start_time { 1.day.ago }
     end
 
+    trait :finished do
+      end_time { 1.day.ago }
+    end
+
+    trait :unpublished do
+      published_at { nil }
+    end
+
+    trait :published do
+      published_at { Time.current }
```

Comment:
As the `published_at` attribute inside the `election` factory is default set to `Time.current`, I think you can remove this trait and just use the `election` factory for tests.

Reply:
I've just deleted it! Thanks!

Reply:
In the rest of modules they do it like that but seems that it's not necessary

Reply:
At the end we deleted `unpublished` and put it as the default value. Thank you Svenja for making me thinking about that 😃